test/util/AllocatorTestHelpers.h

```diff
@@ -6,13 +6,13 @@
 
 #include "global/Id.h"
 #include "util/AllocatorWithLimit.h"
+#include "util/MemorySize/MemorySize.h"
 
 namespace ad_utility::testing {
 // Create an unlimited allocator.
 inline ad_utility::AllocatorWithLimit<Id>& makeAllocator() {
   static ad_utility::AllocatorWithLimit<Id> a{
-      ad_utility::makeAllocationMemoryLeftThreadsafeObject(
-          std::numeric_limits<size_t>::max())};
+      ad_utility::makeAllocationMemoryLeftThreadsafeObject(MemorySize::max())};
```

Comment:
This is the function I was talking about the whole time.

Reply:
I thought you wanted `getAllocator`, not `makeAllocator`?

Reply:
I meant this function, I just wasn't sure about its exact name, so take this one and consistently use it above.

Reply:
Alright, on it.